src/style.css

```diff
@@ -1 +1,60 @@
-/* Styles go here */
+body {
+  margin: 0 auto;
+  padding: 0;
+  font-family: "Roboto", sans-serif;
+}
+
+.main {
+  display: flex;
+  justify-content: space-between;
+  padding: 0 50px;
+}
+
+.logo {
+  display: flex;
+  align-items: center;
+  padding-top: 10px;
```

Comment:
Don't use padding here. Just set height to `a` and `align-items: center;` to header